### DISCUSSION TITLE
Where can I find release notes?

### DISCUSSION THREAD

**Discussion by zhechenghu** [remotely-save/remotely-save#138] [General]
Hi,

Thank you for developing such an excellent plug-in! But I cannot find where is the release note. Could you please tell me?

**Comment by sytone** [comment]
There are comments in the release on what has changed. Look to the left and click on the releases.

**Reply by sytone** [reply]
That would be right... need coffee

**Reply by zhechenghu** [reply]
Wow, thanks! I had just realized there is a release page on the left of the tag page.
